templates/stripes/stripe-toc.php

```diff
@@ -0,0 +1,31 @@
+<?php $toc = $this->tableOfContents($scope) ?>
+
+<?php if (!empty($toc)) : ?>
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <div class="vssl-stripe--card">
+      <?php if (!empty($title['html'])) : ?>
+        <div class="vssl-stripe--toc--title"><?= $this->inline($title['html']) ?></div>
+      <?php endif; ?>
+
+      <?php foreach ($toc as $i => $item) : ?>
+
+        <?php if ($i === 0 || $toc[$i - 1]['level'] < $item['level']) : ?>
+        <ul>
+        <?php endif ?>
+
+          <li>
+            <a href="#<?= $item['id'] ?>">
+              <?= $item['text'] ?>
```

Comment:
Can we name these like `label` or something less generic than `text`? Especially with textblock stripes, etc.

Reply:
what do you think about `'headingText'`?

Reply:
Love it